src/Tokenizer.php

```diff
@@ -857,6 +857,11 @@ private function createNextToken(string $string, ?Token $previous = null): Token
             );
         }
 
+        // PostgreSQL cast operator
+        if (substr($string, 0, 2) === '::') {
+            return new Token(Token::TOKEN_TYPE_BOUNDARY, '::');
+        }
```

Comment:
I think the same can be achieved by adding `::` to this list: https://github.com/doctrine/sql-formatter/blob/21139be16870171c4f067de6d0c700769a22b9a3/src/Tokenizer.php#L713

It has to be added before `:` though.

Reply:
You're right, it works :-)